### PR TITLE
feat: add encryption key to env validator

### PR DESCRIPTION
**Pull request by djabarovgeorge**

### CODE REVIEW

File: apps/api/src/config/env-validator.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ const validators: { [K in keyof any]: ValidatorSpec<any[K]> } = {
   NOVU_API_KEY: str({
     default: '',
   }),
+  STORE_ENCRYPTION_KEY: str(),
```
Comment (scopsy) [inline apps/api/src/config/env-validator.ts:33]: @djabarovgeorge is the 32 length required? If so maybe we can add here a min length validation with envalid

**Review by scopsy — APPROVED** [review]

File: apps/api/src/config/env-validator.ts (hunk 1 above)
Comment (djabarovgeorge) [reply apps/api/src/config/env-validator.ts:33]: @scopsy It is a great idea, I did try to create a custom validator for it but I am not sure it supports passing params in it.

Comment (djabarovgeorge) [reply apps/api/src/config/env-validator.ts:33]: Actually, let me do one dedicated validator for it.

File: apps/api/src/config/env-validator.ts [hunk 2]
```diff
@@ -1,6 +1,13 @@
-import { port, str, url, ValidatorSpec, bool } from 'envalid';
+import { makeValidator, port, str, url, ValidatorSpec } from 'envalid';
 import * as envalid from 'envalid';
 
+const str32 = makeValidator((variable) => {
+  if (!(typeof variable === 'string') || variable.length != 32) {
+    throw new Error('Expected to be string 32 char long');
```
Comment (scopsy) [inline apps/api/src/config/env-validator.ts:6]: @djabarovgeorge does it needs to be **exactly** 32? if so why?

**Review by scopsy — APPROVED** [review]

**Pull request merged**